components/crossplane-control-plane/base/namespace-claim-configmap.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: namespace-claim-configmap
+  namespace: crossplane-control-plane
+data:
+  namespace-claim-cleaner.sh: |
+    #!/bin/bash
```

Comment:
Is this going to fail properly if any of the piped commands below fail? I suspect it won't without `set`ting some options.